.github/workflows/deployments.yml

```
@@ -99,9 +99,13 @@ jobs:
       - name: Determine build info
         id: build_info
         run: |
-          echo "Concurrency group: ${{ github.event.workflow_run.head_branch || github.ref_name }}"
+          if ${{ github.event_name == 'workflow_dispatch' }}; then 
+            defaults="${{ (inputs.update_registry_cache && 'linux/arm64 linux/amd64') || 'linux/amd64,linux/arm64' }}"
+            requested_platforms=`echo "${{ inputs.platforms }}" | tr + , | sed "s#default#$defaults#g"`
```

I don't know if this part was already tested in the test runs, so I'm making this suggestion just in case it hasn't been exercised before.
```suggestion
            requested_platforms=`echo "${{ inputs.platforms }}" | tr + , | sed "s#default#${defaults}#g"`
```